### DISCUSSION TITLE
Lorem ipsum dolor sit amet

### DISCUSSION THREAD

**Discussion by ghost** [pystardust/ytfzf#722] [Q&A] [answered]
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

**Comment by Euro20179** [comment, marked as answer]
> Even if I create script directly in /usr/bin/yt-dlp_to_mpv with yt-dlp -o - $1 | mpv -, and add video_player="yt-dlp_to_mpv" to my ~/.config/ytfzf/conf.sh

overriding video player this way doesn't work, you have one of 2 options
either put
```
video_player () {
    yt-dlp -o - $1 | mpv -
}
```
or
```
load_url_handler "/usr/bin/yt-dlp_to_mpv"
```
in your ytfzf config

The difference in this situation is minor, but basically `video_player` is used by the builtin url handler, whereas `load_url_handler` will override the builtin url handler
